include/swift/ClangImporter/ClangImporter.h

```diff
@@ -178,9 +179,9 @@ class ClangImporter final : public ClangModuleLoader {
   /// Note that this method does no filtering. If it finds the type in a loaded
   /// module, it returns it. This is intended for use in reflection / debugging
   /// contexts where access is not a problem.
-  void lookupRelatedEntity(StringRef clangName, ClangTypeKind kind,
```

Comment:
Why is it okay to drop the "kind" parameter?

Reply:
Because it was not used by the function.

Reply:
This was deliberately an interface designed to handle arbitrary mangled names, even though we're not using the `kind` for anything yet.